Delphi/Project/Source/uInstallLocationFrm.pas

```diff
@@ -60,10 +61,13 @@ function ShowInstallLocationForm(var aInstallInfo: TInstallInfo): TResultStatus;
 end;
 
 procedure TfrmInstallLocation.btnBrowseClick(Sender: TObject);
+var
+  folder: string;
 begin
-  if RzSelectFolderDialog1.Execute then
+  folder := fldLocation.Text;
+  if SelectDirectory('Select Install Location', '', Folder, [sdNewUI, sdNewFolder, sdShowEdit], Self) then
```

Comment:
Please remove `sdShowEdit`.  The edit field is confusing and I don't think necessary.